src/integrationTest/sam.test.ts

```diff
@@ -263,6 +264,13 @@ describe('SAM Integration Tests', async function() {
 
                 it('invokes and attaches on debug request (F5)', async function() {
                     setTestTimeout(this.test?.fullTitle(), 60000)
+                    // Allow previous sessions to go away.
+                    waitUntil(
```

Comment:
Thoughts on making this an after each activity, and asserting that active debug session is undefined before each test?

I realize it isn't applicable to all tests in the suite, but it is no less valid, and codifies an isolated behavior.

Reply:
> and codifies an isolated behavior.

Moving it out of the test seems like the opposite to me, it disconnects the steps.  This is an example of "explicit/obvious is good", imo.

Reply:
My thought was that this accommodates a side effect caused by a different test. The downside to my proposal is some duplication of logic however. Either way is fine by me.